packages/manager/src/features/linodes/LinodesCreate/LinodeCreateContainer.tsx

```diff
@@ -799,6 +811,7 @@ class LinodeCreateContainer extends React.PureComponent<CombinedProps, State> {
             accountBackupsEnabled={getAccountBackupsEnabled()}
             setAuthorizedUsers={this.setAuthorizedUsers}
             updateUserData={this.setUserData}
+            updateSearchParams={this.setSearchParams}
```

Comment:
Do we need to pass this as a prop? Can `<LinodeCreate />` just have its own `setSearchParams` so we don't need another prop?

```suggestion
            updateSearchParams={this.setSearchParams}
```